### ISSUE TITLE
Логические выражения

### ISSUE THREAD

**Issue by viktorovich87** [asd95/tt-adminui#5]
Почему такое выражение 
open ? <DateRange ...  : null

не заменить на 
open && <DateRange...  

первый вариант обычно ставится тогда,  когда есть какое-то событие по одному из условий. Если событие будет выполняться только по одному условию - лучше использовать второй вариант

**Comment by asd95** [comment]
Да, тут вы правы. Просто привык писать как в первом случае, мне кажется она более логичной.
Спасибо, я учту в дальнейшем

**Comment by viktorovich87** [comment]
то что логичной - это понятно. просто запись короче, хотя логика такая же
